integrations.go

```diff
@@ -6,6 +6,16 @@ import (
 	"github.com/Scalingo/go-scalingo/http"
 )
 
+type ScmType string
```

Comment:
```suggestion
type SCMType string
```

Reply:
Your review may be a pain in the ass, but it's relevant to a lot of small points. And that's not so bad :)